papers/IDML/README.md

```diff
@@ -0,0 +1,79 @@
+# On the Effectiveness of Sentence Encoding for Intent Detection Meta-Learning
+
+This repository contains the open-sourced official implementation of the paper
+
+[On the Effectiveness of Sentence Encoding for Intent Detection Meta-Learning](https://openreview.net/pdf?id=SzGx4ZQfHZq) (NAACL 2022).   
+_Tingting Ma, Qianhui Wu, Zhiwei Yu, Tie-jun Zhao and Chin-Yew Lin_
+
+If you find this repo helpful, please cite the following paper
+```tex
+@inproceedings{ma-etal-2022-intentemb,
+    title = {On the Effectiveness of Sentence Encoding for Intent Detection Meta-Learning},
+    author = {Tingting Ma and Qianhui Wu and Zhiwei Yu and Tiejun Zhao and Chin-Yew Lin},
+    booktitle = {xxx},
+    year = {2022},
+    publisher = {xxx},
+    url = {xxx},
+    pages = {xxx}
+}
+```
+
+For any questions/comments, please feel free to open GitHub issues or email the <a href="mailto:hittingtingma@gmail.com">fist author</a> directly.
+
+## Requirements
+
+```
+pip install -r requirements.txt 
+```
+Note that we conduct our experiments with python3.7, and if you want to use this code with python3.9 and numpy 1.21.0, please upgrade the sentencepiece to 0.1.94.

```

Comment:
Please re-write this (and the requirements file) to use the newer version. The text can say "The original experiments were conducted with Python 3.7, numpy ???, and sentencepiece ???; which now have security vulneralibilities. Please try reverting to those versions in case of any problem."

Reply:
It seems the requirements.txt file is correct. No need to change it, just the text in the README.